tests.py

```diff
@@ -12,28 +12,28 @@ def setUp(self):
         self.msg = "test"
 
     def test_run_check(self, mock_sys):
-        with self.assertRaises(NotImplementedError):
-            self.plugin_inst.run_check()
+        self.assertRaises(NotImplementedError,
+                          lambda: self.plugin_inst.run_check())
```

Comment:
why the lambda and not just self.plugin_inst.run_check (without the parenthesis)?